hawkular-inventory-itest/src/test/groovy/org/hawkular/inventory/rest/test/InventoryITest.groovy

```diff
@@ -1291,6 +1292,90 @@ class InventoryITest extends AbstractTestBase {
+                data    : [
+                        id: "sync-feed"
+                ],
+                children: [
+                        resource    : [
+                                [
+                                        data    : [
+                                                id              : "resource",
+                                                resourceTypePath: "resourceType"
+                                        ],
+                                        children: [
+                                                resource: [
+                                                        [
+                                                                data: [
+                                                                        id              : "childResource",
+                                                                        resourceTypePath: "../resourceType"
+                                                                ]
+                                                        ]
+                                                ]
+                                        ]
+                                ]
+                        ],
+                        resourceType: [
+                                [
+                                        data: [
+                                                id  : "resourceType",
+                                                name: "My Resource Type With A Friendly Name"
+                                        ]
+                                ]
+                        ],
+                        metric      : [
+                                [
+                                        data: [
+                                                id                : "metric",
+                                                metricTypePath    : "metricType",
+                                                collectionInterval: 0
+                                        ]
+                                ]
+                        ],
+                        metricType  : [
+                                [
+                                        data: [
+                                                id                : "metricType",
+                                                type              : "GAUGE",
+                                                unit              : "NONE",
+                                                collectionInterval: 0,
+                                                name              : "My Metric Type With A Friendly Name"
+                                        ]
+                                ]
+                        ],
+                ]
+        ]
+
+        try {
+            def response = client.post(path: "$basePath/feeds", body: "{\"id\": \"sync-feed\"}")
+            assertEquals(201, response.status)
+
+            response = client.post(path: "$basePath/sync/f;sync-feed", body: structure)
```

Comment:
+1 the REST api looks pretty easy to use. And for creating the JSON there is the builder in place.